securedrop/source_templates/base.html

```diff
@@ -17,7 +17,7 @@
         {% block header %}
         <div id="header">
           <a href="{% if 'logged_in' in session %}{{ url_for('main.lookup') }}{% else %}{{ url_for('main.index') }}{% endif %}" class="no-bottom-border">
-            <img src="{{ url_for('main.select_logo') }}" class="logo small" alt="SecureDrop" width="250">
+            <img src="{{ url_for('main.select_logo') }}" class="logo small" alt="Logo Image" width="250">
```

Comment:
Consider using  `gettext()` to allow this string to be translated.

Reply:
Made `Logo Image` string available for translation via https://github.com/freedomofpress/securedrop/pull/4980/commits/a9f8b9abdab6845f6bde50575272c12e5849d2b6.